dolphinscheduler-alert/dolphinscheduler-alert-plugins/dolphinscheduler-alert-feishu/src/main/java/org/apache/dolphinscheduler/plugin/alert/feishu/FeiShuSender.java

```diff
@@ -177,41 +178,38 @@ private String sendMsg(AlertData alertData) throws IOException {
 
     static final class FeiShuSendMsgResponse {
```

Comment:
It's better to use `@Data` to avoid getter and setter.

Reply:
I try to make as few changes as possible to avoid failing the code standards.
The original code style is like this.

Reply:
What do you mean by `failing the code standards`?

Reply:
`spotless-maven-plugin` This plugin is giving me a headache.

Reply:
I think following the original code and fixing it is always the most effective way to avoid unknown problems.
Do you need to modify and resubmit? The bug has been fixed and passed the review

Reply:
You should find ways to modify the code while conforming to code style.

Reply:
> The bug has been fixed and passed the review

This is the review process, and it hasn't passed yet.

Reply:
ok